docs/expressions.md

```diff
@@ -331,55 +331,55 @@ is also returned with a value of `1`. Primarily for use with `depends`.
 Example: `alert("host.down", "crit")` returns the crit
 expression from the host.down alert.
 
-## abs(number)
+## abs(numberSet) numberSet
 
-Returns the absolute value of the number.
+Returns the absolute value of each element in the numberSet.
 
-## d(string)
+## d(string) scalar
 
 Returns the number of seconds of the [OpenTSDB duration string](http://opentsdb.net/docs/build/html/user_guide/query/dates.html).
 
-## des(series, alpha, beta)
+## des(series, alpha scalar, beta scalar) series
 
 Returns series smoothed using Holt-Winters double exponential smoothing. Alpha
 (scalar) is the data smoothing factor. Beta (scalar) is the trend smoothing
 factor.
 
-## dropge(series, number)
+## dropge(series, scalar) series
 
 Remove any values greater than or equal to number from a series. Will error if this operation results in an empty series.
 
-## drople(series, number)
+## drople(series, scalar) series
 
 Remove any values lower than or equal to number from a series. Will error if this operation results in an empty series.
 
-## dropna(series)
+## dropna(series) series
 
 Remove any NaN or Inf values from a series. Will error if this operation results in an empty series.
 
-## epoch()
+## epoch() scalar
 
 Returns the Unix epoch in seconds of the expression start time (scalar).
 
-## filter(series, number)
+## filter(series, numberSet) series
 
 Returns all results in series that are a subset of anything in number, or
 that have number as a subset. Useful with the limit and sort functions to
 return the top X results of a query.
 
-## limit(number, count)
+## limit(numberSet, count scalar) numberSet
 
 Returns the first count (scalar) results of number.
 
-## lookup(table, key)
+## lookup(table string, key string) numberSet 
 
 Returns the first key from the given lookup table with matching tags.
 
-## nv(number, scalar)
+## nv(numberSet, scalar) numberSet
 
 Change the NaN value during binary operations (when joining two queries) of unknown groups to the scalar. This is useful to prevent unknown group and other errors from bubbling up.
 
-## sort(number, asc_desc)
+## sort(numberSet, asc_desc string) numberSet
```

Comment:
And I disagree that the description is clear. You should be able to call a function knowing only it's signature, and asc_desc is not a defined type. If you want to do that you should do what .NET and Go do and use enums or consts instead of strings